src/vs/editor/common/model/DecorationProvider.ts

```diff
@@ -0,0 +1,29 @@
+/*---------------------------------------------------------------------------------------------
```

Comment:
I think we typically have lowercase file names.

Reply:
Right, I have to get used to that. Maybe I used the "Move To File" refactoring that names the file as the symbol that is getting moved.